package.json

```diff
@@ -58,7 +58,7 @@
     "@typescript-eslint/parser": "^5.59.5",
     "@zowe/cli": "^7.23.2",
     "@zowe/cli-test-utils": "^7.23.2",
-    "@zowe/imperative": "^5.22.2",
+    "@zowe/imperative": "^8.0.0-next < 9.0.0",
```

Comment:
your initial version string was accurate.
After double-checking on https://semver.npmjs.com/ 
if you do `@zowe/imperative` and `^5.0.0-next` you should only get all 5.x (with prerelease versions included)